ci/start-console.sh

```diff
@@ -4,63 +4,55 @@ set -euo pipefail
 script_dir=$(dirname "$0")
 source ${script_dir}/configure/openshift.sh
 
-CONSOLE_CONTAINER_NAME="okd-console"
-FORKLIFT_NAMESPACE="konveyor-forklift"
-
-BASE_HOST_URL="https://localhost"
-
-PLUGIN_NAME="forklift-console-plugin"
-PLUGIN_URL="http://localhost:9001"
-
-CONTAINER_NETWORK="--network=host"
-
-CONSOLE_IMAGE="quay.io/openshift/origin-console:latest"
-CONSOLE_PORT="9000"
-CONSOLE_PORT_PUBLISH="--publish=$CONSOLE_PORT:$CONSOLE_PORT"
-CONSOLE_CONTAINER_NAME_RUN="--name=$CONSOLE_CONTAINER_NAME"
-
-# On macOS
+# Default values with CLI overrides
+CONSOLE_CONTAINER_NAME="${CONSOLE_CONTAINER_NAME:-okd-console}"
+FORKLIFT_NAMESPACE="${FORKLIFT_NAMESPACE:-konveyor-forklift}"
+BASE_HOST_URL="${BASE_HOST_URL:-https://localhost}"
+PLUGIN_NAME="${PLUGIN_NAME:-forklift-console-plugin}"
+PLUGIN_URL="${PLUGIN_URL:-http://localhost:9001}"
+CONTAINER_NETWORK="${CONTAINER_NETWORK:---network=host}"
+CONSOLE_IMAGE="${CONSOLE_IMAGE:-quay.io/openshift/origin-console:latest}"
+CONSOLE_PORT="${CONSOLE_PORT:-9000}"
+CONSOLE_PORT_PUBLISH="--publish=${CONSOLE_PORT}:${CONSOLE_PORT}"
+CONSOLE_CONTAINER_NAME_RUN="--name=${CONSOLE_CONTAINER_NAME}"
+PULL_POLICY="${PULL_POLICY:-always}"
+
+# Adjust settings for macOS
 if [[ $(uname) = "Darwin" ]]; then
     BASE_HOST_URL="http://host.containers.internal"
     CONTAINER_NETWORK=""
     PLUGIN_URL="$BASE_HOST_URL:9001"
 fi
 
-# Default to localhost
-INVENTORY_SERVER_HOST="$BASE_HOST_URL:30444"
-SERVICES_API_SERVER_HOST="$BASE_HOST_URL:30446"
+# Default API server hosts
+INVENTORY_SERVER_HOST="${INVENTORY_SERVER_HOST:-${BASE_HOST_URL}:30444}"
+SERVICES_API_SERVER_HOST="${SERVICES_API_SERVER_HOST:-${BASE_HOST_URL}:30446}"
 
 # Look for forklift routes
 if oc_available_loggedin; then
     routes=$(oc get routes -A -o template --template='{{range .items}}{{.spec.host}}{{"\n"}}{{end}}' 2>/dev/null || true)
-    INVENTORY_SERVER_HOST="https://$(echo "$routes" | grep forklift-inventory)"
-    SERVICES_API_SERVER_HOST="https://$(echo "$routes" | grep forklift-services)"
+    INVENTORY_SERVER_HOST="https://$(echo "$routes" | grep forklift-inventory || echo "${INVENTORY_SERVER_HOST}")"
+    SERVICES_API_SERVER_HOST="https://$(echo "$routes" | grep forklift-services || echo "${SERVICES_API_SERVER_HOST}")"
```

Comment:
This won't allow overriding the `routes` values with CLI if needed for both SERVICES_API_SERVER_HOST, INVENTORY_SERVER_HOST

Maybe change to something like:
```
...
if oc_available_loggedin; then
   ...
  INVENTORY_SERVER_HOST=" echo "${INVENTORY_SERVER_HOST}"
  || "https://$(echo "$routes" | grep forklift-inventory)  
  || ${BASE_HOST_URL}:30444"
```

and the same for SERVICES_API_SERVER_HOST